src/Catel.Core/Catel.Core.Shared/Collections/Interfaces/Enums/SuspensionMode.cs

```diff
@@ -0,0 +1,29 @@
+// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="FastObservableCollection.cs" company="Catel development team">
+//   Copyright (c) 2008 - 2016 Catel development team. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Catel.Collections
+{
+    /// <summary>
+    /// The suspension mode.
+    /// </summary>
+    public enum SuspensionMode
```

Comment:
If we add "None", then we can get the default ObservableCollection behavior, correct?